run-validations.sh

```diff
@@ -19,7 +29,7 @@ fi
 pushd "${recorderFolder}"
 function finish { popd; }
 trap finish EXIT
-./run-validations.sh $@
+STACK_VERSION=${STACK_VERSION} ./run-validations.sh $@
```

Comment:
We should not need to re-export the variable as it's already in the environment and therefore inherited by child processes.